### DISCUSSION TITLE
Not able to load a locally stored html file.

### DISCUSSION THREAD

**Discussion by TeckPete** [guysoft/FullPageOS#523] [General]
#### What were you doing?
Trying to open a local stored html page 
[Please be as specific as possible here]
I saved my the html file in /var/www/html/index.html and changed the fullpageos.txt to http://var/www/html/index.html
#### What did you expect to happen?
to pull the the locally stored html page
#### What happened instead?
the screen is black. when a put a web url in it works fine
#### Was there an error message displayed? What did it say?
no error
#### Version of FullPageOS?
0.13.0
[Can be found in /etc/fullpageos_version ALWAYS INCLUDE.]

#### Screenshot(s) showing the problem:
![Capture7](https://github.com/guysoft/FullPageOS/assets/35778227/003bb590-f2c3-4017-963d-0498c2d9272f)
![Capture6](https://github.com/guysoft/FullPageOS/assets/35778227/62c12f91-47e1-4114-b1c1-a529a08d2d53)

[If applicable. Always include if unsure or reporting UI issues.]

**Comment by TeckPete** [comment]
@guysoft could this have anything to do with my problem? I'm sure I just missed something stupid.

![Capture65](https://github.com/guysoft/FullPageOS/assets/35778227/b6aa6849-ead0-4392-83ca-1dd6fa95a6ed)

**Comment by Metatron2k2** [comment]
In order to load a local file you would need to use file:// instead of http. Although I am not sure if that resolves you problem. As far as I know chrome disables JS for local files. Therefore you will probaly have to isntall a webserver (apache, nginx lighttp or similar) on the pi

**Comment by guysoft** [comment]
There is an onboard websever, but then you need to use https://localhost

https://en.wikipedia.org/wiki/Localhost

I am closing this because its not a FullPageOS issue.

Makes me think we might need a forum

**Comment by guysoft** [comment]
I am converting this to a discussion, its not an issue with FullPageOS

**Reply by juanmartin** [reply]
Hi, thanks for all your work @guysoft I am trying FullPageOS and it's very straightforward.

I am interested in hosting the site in the same Pi to have it working even when offline, but since it involves js I need a webserver. You mentioned there is one already, is that documented somewhere?
